packages/plugin-loyalties-api/src/models/ScoreCampaigns.ts

```
@@ -247,25 +195,7 @@ export const loadScoreCampaignClass = (models: IModels, subdomain: string) => {
-            );
-          }
-        } else {
-          placeholder = placeholder.replace(
-            `{{ ${attribute} }}`,
-            target[attribute]
-          );
-        }
+        placeholder = resolvePlaceholderValue(target, attribute);
       }
 
       let changeScore = (eval(placeholder) || 0) * Number(currencyRatio) || 0;
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)